.betterer.eslint.config.js

```diff
@@ -10,6 +10,7 @@ const testingLibraryPlugin = require('eslint-plugin-testing-library');
 
 const grafanaConfig = require('@grafana/eslint-config/flat');
 const grafanaPlugin = require('@grafana/eslint-plugin');
+const grafanaI18nPlugin = require('@grafana/i18n/eslint-config');
```

Comment:
note to self: export/import from `/eslint-plugin`, maybe?

Reply:
agree with this